docs/dashboards/introduction.md

```diff
@@ -252,11 +258,18 @@ If you want to revert a dashboard card to its original visualization settings (i
 
 1. Go to your dashboard and click the **pencil icon** to go into edit mode.
 2. Hover over the card (question) that you want to edit.
-3. Click the **palette icon** to bring up the visualization settings for that card.
-4. Click **Reset to defaults**.
+3. Click the **palette icon** to bring up the visualization settings for that card. 
+4. Click **Reset to defaults** (bottom right).
 5. Click **Done** to save the card's visualization settings.
 6. Click **Save** to save the dashboard.
 
+If you don't see the palette icon, you'll need to:
```

Comment:
This sentence makes me think I should _always_ have the palette icon, but this is contradicting the previous sentence "click on the **pencil** icon or **palette** icon (depending on the chart type)"

I honestly don't even know what example use case this is supposed to be addressing